lib/babe/helpers_test.go

```diff
@@ -133,12 +159,209 @@ func NewTestService(t *testing.T, cfg *core.Config) *core.Service {
 	return s
 }
 
-// newDevGenesisWithTrieAndHeader generates test dev genesis, genesis trie and genesis header
-func newDevGenesisWithTrieAndHeader(t *testing.T) (
+func createTestService(t *testing.T, cfg ServiceConfig, genesis genesis.Genesis,
+	genesisTrie trie.Trie, genesisHeader types.Header) *Service {
+	wasmer.DefaultTestLogLvl = 1
+
+	var err error
+
```

Comment:
Remove this and change below `err = dbSrv.Initialise(&genesis, &genesisHeader, &genesisTrie)` to `err := dbSrv.Initialise(&genesis, &genesisHeader, &genesisTrie)`